schema/organization/Person.md

```diff
@@ -0,0 +1,130 @@
+# Person
+
+The `Person` type allows you to provide details about a person such as their given and family names, any honourifix prefix or suffix, and contact details such as an email address. This type of often used to describe the `authors` of an [`Article`](/Article), or other [`CreativeWork`](/CreativeWork).
```

Comment:
I'm not familiar with the word `honourifix`. should it be `honourific prefix or suffix`, or portmanteau of just `honorifix`?

Reply:
Hah, that's a typo...and my new rap star name! :notes: